rpm/SPECS/docker-ce.spec

```diff
@@ -84,6 +84,10 @@ export DOCKER_GITCOMMIT=%{_gitcommit_engine}
 mkdir -p /go/src/github.com/docker
 ln -s ${RPM_BUILD_DIR}/src/engine /go/src/github.com/docker/docker
 
+%if 1%{?_with_btrfs}
+export DOCKER_BUILDTAGS='exclude_graphdriver_btrfs'
```

Comment:
Looks like this will overwrite the existing `DOCKER_BUILDTAGS` ? https://github.com/docker/docker-ce-packaging/blob/1c4fdf511936510a893a9c1c092191bed3e0fb95/rpm/centos-8/Dockerfile#L14

Reply:
Hi @thaJeztah thanks for reviewing my PR.  I think we can append the DOCKER_BUILDTAGS.
I will give a patch v2.